packages/frontend/src/components/TestSubstep/index.tsx

```diff
@@ -142,7 +143,9 @@ function TestSubstep(props: TestSubstepProps): React.ReactElement {
               </AlertTitle>
 
               <Box sx={{ fontWeight: 400 }}>
-                {formatMessage('flowEditor.noTestDataMessage')}
```

Comment:
This is the only place where `flowEditor.noTestDataMessage` is referenced; so if you remove this we don't need it in our localization `en.json` file anymore.

But don't remove it, I think we can re-use. I will suggest some changes.